pycir/__init__.py

```diff
@@ -0,0 +1 @@
+from .cir import cirPAVA
```

Comment:
This is how you import functions from multiple files into one package. Usage would then be:

```
import pycir

regression = pycir.cirPAVA(x, y, wt)
```